### ISSUE TITLE
Duplicate IDs in layer's Style control

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#508]
Due to duplicate ids on the `input` elements and `for=""` id references in `label` elements one input in https://maps4html.org/experiments/datacube/ is labelled <q>WMS GetMap</q> twice and the other is not labelled.

(See "Name" under "Computed Properties" in the Accessibility tab in the screenshots)

![layer-1](https://user-images.githubusercontent.com/26493779/128942613-0e5cce82-76ec-4669-b650-25943273c3ef.png)
![layer-2](https://user-images.githubusercontent.com/26493779/128942615-83515cfb-d294-42f5-bccb-f58d6ddf3f34.png)
